### DISCUSSION TITLE
Is there a way to use this library offline?

### DISCUSSION THREAD

**Discussion by frostydev99** [uber/react-map-gl#1741] [Q&A]
I am writing a software that will not be able to have internet connection while being used, how can I use MapBox offline manager with this api?

**Comment by louislefebvre** [comment]
I am in the same situation as you, I could not find anything about having mapbox-gl tiles downloaded, cached, stored, or something else for an offline use.

**Comment by Pessimistress** [comment]
> how can I use MapBox offline manager

This library is only a wrapper of mapbox-gl. You need to ask the Mapbox GL JS repo regarding any service-specific features.
